### ISSUE TITLE
Add Tarantool to OpenWRT

### ISSUE THREAD

**Issue by rtsisyk** [tarantool/tarantool#1698]
Add a Makefile for Tarantool to OpenWRT project.

https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md

**Comment by zloidemon** [comment]
Yeah, a lot MIPS processors, btw need to use quemu with mips 64 bit emulation. FreeBSD has automation of process to build packages on arm/mips via [poudriere].

[poudriere]: http://www.slideshare.net/eurobsdcon/sean-bruno-building-packages-through-emulation
